Assets/Scripts3D/Generator3D.cs

```diff
@@ -235,21 +244,40 @@ class Room {
         }
     }
 
-    void PlaceCube(Vector3Int location, Vector3Int size, Material material) {
+    void PlaceCube(Vector3Int location, Vector3Int size, Material material, string tag) 
+    {
+        GameObject go = Instantiate(cubePrefab, location, Quaternion.identity);
+        go.GetComponent<Transform>().localScale = size;
+        go.GetComponent<MeshRenderer>().material = material;
+        go.tag = tag;
+    }
+
+    void PlaceCorridorPrefab(Vector3Int location, Vector3Int size, Material material) 
+    {
+        GameObject go = Instantiate(cubePrefab, location, Quaternion.identity);
+        go.GetComponent<Transform>().localScale = size;
+        go.GetComponent<MeshRenderer>().material = material;
+    }
+
+    void StairsPrefab(Vector3Int location, Vector3Int size, Material material, string tag) 
+    {
         GameObject go = Instantiate(cubePrefab, location, Quaternion.identity);
```

Comment:
should it be like this?
```suggestion
        GameObject go = Instantiate(stairsPrefab, location, Quaternion.identity);
```

Reply:
Nah lets keep is as this for now

Its going to instantiate a cube with a "stairs" tag which Ill use to replace with stairs whilst looping through the array  in LevelAssetGenerator (starting on this in next PR)